dashboard/Home.py

```diff
@@ -0,0 +1,31 @@
+"""Main page of the dashboard."""
+
+
+from dotenv import load_dotenv
+import streamlit as st
+
+from database import get_connection, get_popular_tracks, get_popular_albums, get_popular_artists, get_sales_by_tag, get_sales_by_country, get_album_sales_by_album, get_all_album_titles
```

Comment:
It might be easier to just do import *. It'll also be easier to distinguish in the main function where functions are coming from.

Reply:
Cool, instead I've just imported the files